repository.py

```diff
@@ -791,28 +791,36 @@ def installFromYum(targets, mounts, progress_callback, cachedir):
         count = 0
         total = 0
         verify_count = 0
+        progressLine = re.compile('.*(\d+)/(\d+)$')
         while True:
             line = p.stdout.readline()
             if not line:
                 break
             line = line.rstrip()
             logger.log("DNF: %s" % line)
+            norm_line = ' '.join(line.split())
             if line == 'Resolving Dependencies':
                 progress_callback(1)
-            elif line == 'Dependencies Resolved':
+            elif line == 'Dependencies Resolved' or line == 'Dependencies resolved.':
                 progress_callback(3)
             elif line.startswith('-----------------------------------------'):
                 progress_callback(7)
             elif line == 'Running transaction':
                 progress_callback(10)
             elif line.endswith(' will be installed') or line.endswith(' will be updated'):
```

Comment:
Does this ever match now or is setting of `total` exclusively handled by the regexp match below?

Reply:
I kept Yum compatibility. Yes, on one side it can be confusing. On the other it makes it easier to move code from one branch to the other. Not sure how much as the `master` code is just using `dnf`.